### PR TITLE
Flow errors

### PR DESCRIPTION
**Pull request by cloudify**
Please check and try to fix these Flow errors

### CODE REVIEW

File: src/controllers/authenticationController.js [hunk 1]
```diff
@@ -123,6 +124,7 @@ export default class AuthenticationController {
    * @param res
    */
   metadata(req: express$Request, res: express$Response) {
+    // $FlowFixMe
```
Comment (cloudify) [inline src/controllers/authenticationController.js]: what's the problem here?

File: src/controllers/authenticationController.js [hunk 2]
```diff
@@ -91,6 +91,7 @@ export default class AuthenticationController {
         req.query = {};
         req.query.entityID = user.spid_idp;
 
+        // $FlowFixMe
```
Comment (cloudify) [inline src/controllers/authenticationController.js]: what's the problem here?

Comment (lussoluca) [reply src/controllers/authenticationController.js]: Actually our spid-passport package isn't covered by a flow-typed definition, we will have to create a new one.

File: src/controllers/authenticationController.js (hunk 1 above)
Comment (lussoluca) [reply src/controllers/authenticationController.js]: Actually our spid-passport package isn't covered by a flow-typed definition, we will have to create a new one.

**Pull request closed without merging**